EMS/admin-ui-bundle/src/Resources/views/bootstrap5/form/forms.html.twig

```diff
@@ -347,7 +353,7 @@
                                         {{ block(column.tableDataBlock()) }}
                                     {% endfor %}
                                     {% if table.itemActions|length > 0 %}
-                                        <td data-search="">
```

Comment:
This is to exclude those cells from the datatable search. Otherwise we can search on the action's labels

Reply:
This cost me a lot off time but, it's now correctly defined in the column def to not make the column searchable and orderable. It was not working for query datatable with serverside rendering. And throwing a confusing datatable js exception 😒😃

Reply:
It was done in this commit: https://github.com/ems-project/elasticms/pull/978/commits/d7204e45e59fa8b2fc8d9e5820def6a3f65c5440